apps/rotm/models/image-upload.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const url = require('url');
+const Model = require('hof-model');
+const config = require('../../../config');
+
+module.exports = class UploadModel extends Model {
+  save() {
+    return new Promise((resolve, reject) => {
+      const attributes = {
+        url: config.upload.hostname
```

Comment:
is this hitting firearms support document url? when I was debugging that's what it was evaluating to

Reply:
It uses the following config. If you have `FILE_VAULT_URL` set to the firearms supporting document url in the environment that you were debugging it in, then that is entirely possible.

Where were you debugging?

```
 upload: {
     maxFileSize: '100mb',
     hostname: process.env.FILE_VAULT_URL ?
       process.env.FILE_VAULT_URL :
      `http://${localhost()}/api/image-upload`
 
   },
```

Reply:
yeah I forgot that my environment variable is set in my bash_profile to firearms, so it's probably that